db/springbootapp_db_script.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS employee;
+CREATE TABLE IF NOT EXISTS employee (
+  employee_id int(11) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  first_name varchar(250) NOT NULL,
+  last_name varchar(250) NOT NULL,
+  added_date varchar(250) NOT NULL,
+  employment_status BIT(1) NOT NULL --0 for deleted record / 1 for valid record,
```

Comment:
Tiny modification, put a space after the `--` or else the db will not recognize it as a comment.